compose/project.py

```diff
@@ -54,7 +54,7 @@ def visit(n):
     return sorted_services
 
 
-class Project(object):
+class Project(ParallelContainerManager):
```

Comment:
I don't understand what your real concern is here? You want to avoid inheritance but not giving me any specific reasons why/what problems you can see. Mixins are a common pattern and a good first step to pull this functionality out as we explore further cases of parallelisification. Help me to understand what your intent is here.

This was a more iterative approach of make it work first, refactor it, improve. So the style of passing in services is a reflection of those steps. `stop` and `kill` have the same logic for creating the container list, so I'd rather not duplicate that logic in project.py when I can have it once in here with a single code branch for remove, which is a different flow case. Granted, at the time I had `restart` as well but then had to back that out.